packages/teams-js/src/internal/videoEffectsUtils.ts

```diff
@@ -219,7 +219,10 @@ class OneTextureHeader {
   private readonly ONE_TEXTURE_INPUT_ID = 0x6f746931;
   private readonly INVALID_HEADER_ERROR = 'Invalid video frame header';
   private readonly UNSUPPORTED_LAYOUT_ERROR = 'Unsupported texture layout';
-  public constructor(private readonly headerBuffer: ArrayBuffer, private readonly notifyError: (string) => void) {
+  public constructor(
+    private readonly headerBuffer: ArrayBuffer,
+    private readonly notifyError: (string) => void,
+  ) {
```

Comment:
All the changes in this file were automatically made by prettier. They aren't technically related to my change but since they are innocuous I left them in (and they would keep coming back in future changes until they were made, anyway)